### ISSUE TITLE
d problemos - maybe cause of missing clock_getres - ultra low priority

### ISSUE THREAD

**Issue by eyberg** [nanovms/nanos#391]
```
eyberg@s1:~/dz$ ops run -p 8083 -f -d -c config.json bob
/tmp/resolv=>/etc/resolv.conf
/tmp/hostname=>/proc/sys/kernel/hostname
bob.o=>bob.o
bob=>/bob
Finding dependent shared libs
sdf
sdf2
sdf3
sdf4
booting image ...
warning: TCG doesn't support requested feature: CPUID.01H:ECX.vmx [bit 5]
assigned: 10.0.2.15
read program complete: 00000020000000000 (children:(bob.o:(extents:(0:(offset:524800 length:18036)) filelength:18036) kernel:(extents:(0:(offset:543232 length:1438856)) filelength:1438856) proc:(children:
(sys:(children:(kernel:(children:(hostname:(extents:(0:(offset:524288 length:7)) filelength:7))))))) bob:(extents:(0:(offset:4278272 length:942344)) filelength:942344) lib64:(children:(ld-linux-x86-64.so.
2:(extents:(0:(offset:4115456 length:162632)) filelength:162632))) dev:(children:(urandom:(special:;;) null:(special:;;;))) etc:(children:(resolv.conf:(extents:(0:(offset:4114944 length:18)) filelength:18
))) lib:(children:(x86_64-linux-gnu:(children:(librt.so.1:(extents:(0:(offset:4083200 length:31712)) filelength:31712) libpthread.so.0:(extents:(0:(offset:3944448 length:138696)) filelength:138696) libgcc
_s.so.1:(extents:(0:(offset:1982464 length:92552)) filelength:92552) libc.so.6:(extents:(0:(offset:2075136 length:1868984)) filelength:1868984)))))) futex_trace:t program:/bob trace:t fault:t debugsyscall
s:t environment:() arguments:())
 1 run frame 00000000100200c00, RIP=00000000500000c30
 1 brk
 1 uname
 1 access
 1 access
 1 open
 1 open: "/etc/ld.so.cache", flags 80000, mode 1
 1 "/etc/ld.so.cache" - not found
 1 open
 1 open: "/lib/x86_64-linux-gnu/tls/libpthread.so.0", flags 80000, mode 227168
 1 "/lib/x86_64-linux-gnu/tls/libpthread.so.0" - not found
 1 stat
 1 open
 1 open: "/lib/x86_64-linux-gnu/libpthread.so.0", flags 80000, mode 227168
 1    fd 3, file length 138696
 1 read
1 file_read: (extents:(0:(offset:3944448 length:138696)) filelength:138696), dest 0000000010163f740, length 832, offset 0 (infinity), file length 138696
 1 sleep
 1 file_op_complete: len 832, status (none) (OK)
 1 wakeup 1->1 0000000050001b387
 1 run frame 00000000100200c00, RIP=0000000050001b387
 1 fstat
 1 fd 3, stat 0000000010163f5d8
 1 st_ino 2000002f000, st_mode 81a4, st_size 21dc8
 1 mmap
 1 mmap: target 00000000000000000, size 21c428, prot 5, flags 802, fd 3, offset 0
 1   read file, blocking...
 1 wakeup 1->1 0000000050001b4ba
 1 run frame 00000000100200c00, RIP=0000000050001b4ba
 1 mprotect
 1 mmap
 1 mmap: target 00000007000217000, size 2000, prot 3, flags 812, fd 3, offset 17000
 1    fixed at 00000007000217000
 1   read file, blocking...
 1 wakeup 1->1 0000000050001b4ba
 1 run frame 00000000100200c00, RIP=0000000050001b4ba
 1 mmap
 1 mmap: target 00000007000219000, size 3428, prot 3, flags 32, fd 18446744073709551615, offset 0
 1    fixed at 00000007000219000
 1    anon target: 7000219000, len: 4000 (given size: 3428)
 1 close
 1 open
 1 open: "/lib/x86_64-linux-gnu/librt.so.1", flags 80000, mode 227168
 1    fd 3, file length 31712
 1 read
 1 file_read: (extents:(0:(offset:4083200 length:31712)) filelength:31712), dest 0000000010163f710, length 832, offset 0 (infinity), file length 31712
 1 sleep
 1 file_op_complete: len 832, status (none) (OK)
 1 wakeup 1->1 0000000050001b387
 1 run frame 00000000100200c00, RIP=0000000050001b387
 1 fstat
 1 fd 3, stat 0000000010163f5a8
 1 st_ino 20000032000, st_mode 81a4, st_size 7be0
 1 mmap
 1 mmap: target 00000000000000000, size 1000, prot 3, flags 22, fd 18446744073709551615, offset 0
 1    anon target: 7100000000, len: 1000 (given size: 1000)
 1 mmap
 1 mmap: target 00000000000000000, size 207bc0, prot 5, flags 802, fd 3, offset 0
 1   read file, blocking...
 1 wakeup 1->1 0000000050001b4ba
 1 run frame 00000000100200c00, RIP=0000000050001b4ba
 1 mprotect
 1 mmap
 1 mmap: target 00000007200206000, size 2000, prot 3, flags 812, fd 3, offset 6000
1    fixed at 00000007200206000
 1   read file, blocking...
 1 wakeup 1->1 0000000050001b4ba
 1 run frame 00000000100200c00, RIP=0000000050001b4ba
 1 close
 1 open
 1 open: "/lib/x86_64-linux-gnu/libgcc_s.so.1", flags 80000, mode 227168
 1    fd 3, file length 92552
 1 read
 1 file_read: (extents:(0:(offset:1982464 length:92552)) filelength:92552), dest 0000000010163f6e0, length 832, offset 0 (infinity), file length 92552
 1 sleep
 1 file_op_complete: len 832, status (none) (OK)
 1 wakeup 1->1 0000000050001b387
 1 run frame 00000000100200c00, RIP=0000000050001b387
 1 fstat
 1 fd 3, stat 0000000010163f578
 1 st_ino 20000029000, st_mode 81a4, st_size 16988
 1 mmap
 1 mmap: target 00000000000000000, size 216430, prot 5, flags 802, fd 3, offset 0
 1   read file, blocking...
 1 wakeup 1->1 0000000050001b4ba
 1 run frame 00000000100200c00, RIP=0000000050001b4ba
 1 mprotect
 1 mmap
 1 mmap: target 00000007300215000, size 2000, prot 3, flags 812, fd 3, offset 15000
 1    fixed at 00000007300215000
 1   read file, blocking...
 1 wakeup 1->1 0000000050001b4ba
 1 run frame 00000000100200c00, RIP=0000000050001b4ba
 1 close
 1 open
 1 open: "/lib/x86_64-linux-gnu/libc.so.6", flags 80000, mode 227168
 1    fd 3, file length 1868984
 1 read
 1 file_read: (extents:(0:(offset:2075136 length:1868984)) filelength:1868984), dest 0000000010163f6b0, length 832, offset 0 (infinity), file length 1868984
 1 sleep
 1 file_op_complete: len 832, status (none) (OK)
 1 wakeup 1->1 0000000050001b387
 1 run frame 00000000100200c00, RIP=0000000050001b387
 1 fstat
 1 fd 3, stat 0000000010163f548
 1 st_ino 2000002c000, st_mode 81a4, st_size 1c84b8
 1 mmap
 1 mmap: target 00000000000000000, size 3c99a0, prot 5, flags 802, fd 3, offset 0
 1   read file, blocking...
 1 wakeup 1->1 0000000050001b4ba
 1 run frame 00000000100200c00, RIP=0000000050001b4ba
1 mprotect                                                                                                                                                                                        [23/1051]
 1 mmap
 1 mmap: target 000000074003c0000, size 6000, prot 3, flags 812, fd 3, offset 1c0000
 1    fixed at 000000074003c0000
 1   read file, blocking...
 1 wakeup 1->1 0000000050001b4ba
 1 run frame 00000000100200c00, RIP=0000000050001b4ba
 1 mmap
 1 mmap: target 000000074003c6000, size 39a0, prot 3, flags 32, fd 18446744073709551615, offset 0
 1    fixed at 000000074003c6000
 1    anon target: 74003c6000, len: 4000 (given size: 39a0)
 1 close
 1 mmap
 1 mmap: target 00000000000000000, size 1000, prot 3, flags 22, fd 18446744073709551615, offset 0
 1    anon target: 7500000000, len: 1000 (given size: 1000)
 1 mmap
 1 mmap: target 00000000000000000, size 2000, prot 3, flags 22, fd 18446744073709551615, offset 0
 1    anon target: 7600000000, len: 2000 (given size: 2000)
 1 arch_prctl
 1 mprotect
 1 mprotect
 1 mprotect
 1 mprotect
 1 mprotect
 1 mprotect
 1 set_tid_address
 1 set_robust_list
nosyscall set_robust_list
 1 rt_sigaction
 1 rt_sigaction
 1 rt_sigprocmask
 1 getrlimit
 1 brk
 1 brk
 1 clock_getres
 1 clock_getres
 1 clock_getres
 1 clock_getres
 1 clock_getres
 1 clock_getres
 1 clock_getres
 1 rt_sigaction
 1 rt_sigaction
 1 open
 1 open: "/proc/self/maps", flags 80000, mode 1b6
 1 "/proc/self/maps" - not found
 1 clock_getres
Divide by 0
interrupt: 0000000000000000
frame: 0000000100200c00
rax: 000000003b9aca00
rbx: 0000000000000001
rcx: 00000000c0000100
rdx: 0000000000000000
rsi: 000000010163fc30
rdi: 0000000000000001
rbp: 000000010163fc50
rsp: 000000010163fc20
r8: 0000000000000020
r9: 0000000000000040
r10: 000000000069fc60
r11: 0000000000000000
r12: 0000000000690a70
r13: 000000000046a6c4
r14: 00000000006a1a30
r15: 0000000000000003
rip: 000000000046a60e
flags: 0000000000000097
stack trace:
starting gdb
 1 sleep
qemu-system-x86_64: terminating on signal 2

eyberg@s1:~/dz$ cat config.json
{
      "Files": ["bob.o"]
}
eyberg@s1:~/dz$ cat bob.d
import std.stdio;

void main()
{
    writeln("Hello World!");
}
```

**Comment by eyberg** [comment]
https://github.com/nanovms/nanos/issues/1072

**Comment by francescolavra** [comment]
Problem fixed by #1727.
